test/Microsoft.AspNetCore.Mvc.Core.Test/ControllerBaseTest.cs

```diff
@@ -765,7 +765,7 @@ public void HttpUnauthorized_SetsStatusCode()
             var controller = new TestableController();
 
             // Act
-            var result = controller.HttpUnauthorized();
+            var result = controller.Unauthorized();
```

Comment:
Not too concerned about these. I think the test names / URLs are pretty clear either way, no?